jupyter_server_proxy/handlers.py

```diff
@@ -363,7 +364,85 @@ async def proxy(self, host, port, proxied_path):
                 body = b""
             else:
                 body = None
+        accept_type = self.request.headers.get('Accept')
+        if accept_type == 'text/event-stream':
```

Comment:
Add a comment here on *why* this is needed

Reply:
Fixed